cli/docs/cli.json

```diff
@@ -4347,6 +4347,73 @@
                     }
                   ]
                 },
+                {
+                  "name": "auth",
+                  "args": [
+                    {
+                      "long": "authstring",
+                      "help": "Authorization string"
+                    },
+                    {
+                      "long": "peer",
+                      "help": "Peer to apply allow list to"
```

Comment:
Now the help looks like:
```
$ oxide system networking bgp auth --help
Usage: oxide system networking bgp auth [OPTIONS] --rack <RACK> --switch <SWITCH> --port <PORT> --peer <PEER>

Options:
      --rack <RACK>              Id of the rack to add the auth config to
      --profile <PROFILE>        Configuration profile to use for commands
      --switch <SWITCH>          Switch to add the auth config to [possible values: switch0, switch1]
      --port <PORT>              Port to add the auth config to [possible values: qsfp0, qsfp1, qsfp2, qsfp3, qsfp4, qsfp5, qsfp6, qsfp7, qsfp8, qsfp9, qsfp10, qsfp11, qsfp12, qsfp13, qsfp14, qsfp15, qsfp16, qsfp17, qsfp18, qsfp19, qsfp20, qsfp21, qsfp22, qsfp23, qsfp24, qsfp25, qsfp26, qsfp27, qsfp28, qsfp29, qsfp30, qsfp31]
      --peer <PEER>              Peer to apply allow list to
      --authstring <AUTHSTRING>  Authorization string
  -h, --help                     Print help
```

For {`--rack`, `--switch`, `--port`}, it says "auth config", but for `peer`, it says "allow list".

Reply:
Fixed